src-test/org/etools/j1939_84/controllers/part05/Part05Step06ControllerTest.java

```diff
@@ -148,7 +148,9 @@ public void testHappyPathNoFailures() {
         var dm20Packet = new DM20MonitorPerformanceRatioPacket(Packet.create(PGN, 0x00, data));
         when(diagnosticMessageModule.requestDM20(any(), eq(0))).thenReturn(BusResult.of(dm20Packet));
 
-        dataRepository.putObdModule(new OBDModuleInformation(0));
+        OBDModuleInformation obdModuleInformation = new OBDModuleInformation(0);
```

Comment:
This test should be updated to also have a module that doesn't have a DM20 packet stored in part1.